app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -63,6 +57,68 @@ object ProfileHelper {
 		}
 	}
 
+	val deviceAVCCodecProfile by lazy {
+		buildList {
+			when {
+				!MediaTest.supportsAVC() -> {
+					Timber.i("*** Does NOT support AVC")
+					add(
+						ProfileCondition(
+							ProfileConditionType.Equals,
+							ProfileConditionValue.VideoProfile,
+							"none"
+						)
+					)
+				}
+				!MediaTest.supportsAVCHigh10() -> {
+					Timber.i("*** Does NOT support AVC High 10 bit")
+					add(
+						ProfileCondition(
+							ProfileConditionType.NotEquals,
```

Comment:
One concern I have is that now all profiles are accepted rather than just "high", "main", "baseline", "constrained baseline" and optionally "high 10" as was before this change. Are other AVC profiles likely to be hosted on a Jellyfin server? I don't know the answer to that.

How is this function supposed to improve upon the previous `h264VideoProfileCondition` function? That one only accepted that limited set of profiles and added "high 10" when supported. This one supports all profiles if "high 10" is supported, and all profiles except "high 10" when not.